### PR TITLE
New teacher email

### PR DESCRIPTION
**Pull request by ewjordan**
Currently behind a pagemode flag, navigate to https://staging.code.org/private/pm/send_new_teacher_email to set the cookie before creating an account.

Sends an email to new teachers upon account creation:
screen shot 2017-07-21 at 6 46 59 pm

### CODE REVIEW

File: dashboard/app/mailers/teacher_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class TeacherMailer < ActionMailer::Base
+  default from: 'Hadi Partovi <hadi_partovi@code.org>'
+
+  # Email a parent when a student upgrades to a username/password account by adding a parent email
```
Comment (aoby) [inline dashboard/app/mailers/teacher_mailer.rb]: This comment doesn't fit. Copy/paste mixup?

**Comment by aoby** [comment]
Some tests are failing that look unrelated. You might want to merge latest staging and try to get a green test run.

File: dashboard/app/mailers/teacher_mailer.rb (hunk 1 above)
Comment (ewjordan) [reply dashboard/app/mailers/teacher_mailer.rb]: Good catch!

**Comment by ewjordan** [comment]
Still having trouble getting a green run, I'm going to go ahead and merge to staging so I can at least get manual testing done (and actually make sure the email sends without a hitch and looks correct); I'll be kicking off a DTT very soon afterwards anyways, which should catch anything, in which case I'll roll back the merge.

**Pull request merged**